exampleSite/config.yaml

```diff
@@ -171,51 +172,54 @@ Languages:
-        - name: Installation
+        - name: 1. About
           parent: pages
           weight: 10
-          url: installation
-        - name: Billing & Pricing
+          url: about
+        - name: 2. Features
           parent: pages
           weight: 20
-          url: billing-pricing
-        - name: Features
+          url: features
+        - name: 3. Installation
           parent: pages
           weight: 30
-          url: features
+          url: install
+        - name: 4. Contributing
+          parent: pages
+          weight: 50
+          url: contributing
 
-############################# français / French #############################
-  fr:
-    languageName: fr
-    languageCode: fr-fr
+############################# español / Spanish #############################
+  es:
+    languageName: es
+    languageCode: es-mx
     weight: 2
-    home: Page d'accueil
-    copyright: "Copyright &copy; 2020-2022 un thème Hugo par [UNICEF Office of Innovation](https://www.unicef.org/innovation/)"
+    home: Página principal
+    copyright: "Copyright &copy; 2020-2022 una plantilla de [UNICEF Office of Innovation](https://www.unicef.org/innovation/)"
     params:
       banner:
-        title: Comment pouvons nous aider?
-        subtitle: Trouvez rapidement des conseils et des réponses auprès de notre équipe d'assistance ou contactez-nous.
+        title: Plantilla de UNICEF Inventory
+        subtitle: Documentación y un sitio de ejemplo de la plantilla de UNICEF Inventory. Construido para Hugo, un generador de sitios estáticos.
         bg_image: images/banner.jpg
-        placeholder: Avoir une question? Il suffit de demander ici ou de saisir les termes.
+        placeholder: ¿Ud. tiene una pregunta? Usted busca el sitio aquí.
       cta:
         enable: true
-        title: Vous n'avez pas trouvé de réponse à votre question?
-        content: Contactez-nous pour plus de détails sur les services supplémentaires et la tarification du travail personnalisé.
+        title: Integración de Formspree
+        content: Cree un formulario de contacto rápido y fácil para que los visitantes del sitio se comuniquen con la integración integrada de Formspree.
         button:
           enable: true
-          label: Contactez nous
+          label: Ver el formulario
           link: contact
+      footer:
+        description: Documentación de la plantilla de UNICEF Inventory y sitio de ejemplo. Desarrollado por Hugo, un generador de sitio estático. Una herramienta de open source de la Oficina de Innovación de UNICEF, Ventures Team.
     # --- navigation menu ---
     menu:
       main:
-        - name: Accueil
-          weight: 10
-          url: .
         - name: FAQ
-          weight: 20
+          weight: 10
           url: faq
-        - name: Contacter
-          weight: 30
+        - name: Contactar
+          weight: 20
           url: contact
 
         - name: pages
```

Comment:
Same goes here ( line breaks)

Reply:
Addressed in commit bea853a3c63fbd76a36803126fd5336fd5cdd9ad.